src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -257,5 +256,10 @@ private string ResolveContentRootPath(string contentRootPath, string basePath)
             }
             return Path.Combine(Path.GetFullPath(basePath), contentRootPath);
         }
+
+        public void Dispose()
+        {
+            _loggerFactory?.Dispose();
```

Comment:
No, Builder creates it and will reuse for any number of WebHosts you build using it.